spec/features/idv/doc_auth/socure_document_capture_spec.rb

```diff
@@ -293,6 +293,11 @@
           docv_transaction_token: @docv_transaction_token,
         )
 
+        # Confirm that we end up on the Socure page even if we try to
+        # go to the LN / Mock one.
+        visit idv_document_capture_url
+        expect(page).to have_current_path(idv_socure_document_capture_url)
```

Comment:
While this does test the redirect logic and avoids the cost of another run through the feature spec, it seems somewhat buried/hidden when included in
```
  it 'does not track state if state tracking is disabled' do
    ...
```
I guess we don't really have a test that is specific to the generic happy path?